engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -264,6 +264,19 @@ paths:
           description: >
             Returns only workflows with the specified workflow id.  If specified multiple times,
             returns workflows with any of the specified workflow ids.
+        - name: label
+          required: false
+          in: query
+          type: array
+          items:
+            type: string
+          collectionFormat: multi
+          pattern: ^([a-z][-a-z0-9]*[a-z0-9])?[:]([a-z][-a-z0-9]*[a-z0-9])?$
+          description: >
+            Returns workflows with the specified label keys.  If specified multiple times,
```

Comment:
Is it the case that this supports key search and key:value search but not both types in the same query?